js/scripts.js

```diff
@@ -0,0 +1,12 @@
+var list = document.getElementById('list');
+var add = document.getElementById('addElem');
+var element = document.createElement('click','li')[0];
+var length = list.getElementsByTagName('li');
+
+addElem.addEventListner('click', create(addElem));
+
+function create() {
+	element;
```

Comment:
here you need to create this new element - `var element = document.createElement('li')`